nflow-engine/src/main/resources/scripts/db/update-5.5.0-x/db2.update.ddl.sql

```diff
@@ -1 +1,4 @@
 alter table nflow_workflow add started timestamp(3);
+
+update nflow_workflow w, (select workflow_id, min(execution_start) as started from nflow_workflow_action group by workflow_id) a
+  set w.started = a.started where w.id = a.workflow_id
```

Comment:
Semicolon missing from the end of the command.

Reply:
Added